NOTICE

```diff
@@ -1,98 +1,13 @@
-
-=========================================================================
-
-Buildout
-http://www.buildout.org
-Copyright (c) 2006 Zope Foundation and Contributors.
-License: Zope Public License, Version 2.1 (ZPL)
-
-=========================================================================
-
-RequireJS
-http://requirejs.org
-Copyright (c) 2010-2014, The Dojo Foundation
-License: BSD-3-Clause (http://opensource.org/licenses/BSD-3-Clause)
-
-=========================================================================
-
-jQuery
-http://jquery.com/
-Copyright 2014 jQuery Foundation and other contributors
-License: MIT (http://opensource.org/licenses/MIT)
-
-=========================================================================
-
-Underscore.js
-http://underscorejs.org
-Copyright (c) 2009-2014 Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors
-License: MIT (http://opensource.org/licenses/MIT)
-
-=========================================================================
-
-Bootstrap
-http://getbootstrap.com
-Copyright (c) 2011-2014 Twitter, Inc
-License: MIT (http://opensource.org/licenses/MIT)
-Documentation License: Attribution 3.0 Unported (https://creativecommons.org/licenses/by/3.0/)
-
-=========================================================================
-
-Font Awesome
-http://fontawesome.io
-Copyright Dave Gandy
-License: MIT (http://opensource.org/licenses/MIT)
-Font License: SIL OFL 1.1 (URL: http://scripts.sil.org/OFL)
-Documentation License: CC BY 3.0 (URL: http://creativecommons.org/licenses/by/3.0/)
-
-=========================================================================
-
-Ladda
-http://lab.hakim.se/ladda/
-Copyright (C) 2013 Hakim El Hattab, http://hakim.se
-License: MIT (http://opensource.org/licenses/MIT)
-
-=========================================================================
-
-d3.js
-http://d3js.org
-Copyright (c) 2012, Michael Bostock
-All rights reserved.
-License: https://github.com/mbostock/d3/blob/master/LICENSE
-
-=========================================================================
-
-nvd3.js
-http://nvd3.org
-Copyright (c) 2011-2014 Novus Partners, Inc., http://www.novus.com
-License: Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
-
-=========================================================================
-
-angularjs-nvd3-directives
-http://angularjs-nvd3-directives.github.io/angularjs-nvd3-directives/
-Copyright (c) 2013 Christian Maurer
-License: Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
-
-=========================================================================
+Licensed to CRATE Technology GmbH (referred to in this notice as "Crate")
+under one or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information regarding copyright
+ownership.
 
+Crate licenses this software to you under the Apache License, Version 2.0.
```

Comment:
i prefer to have this part as top of the LICENSE file - above the standardized apache license